From c263360d4f40beb2badd04e5bd084f46fd25595b Mon Sep 17 00:00:00 2001
From: Xiangyu Chen <xiangyu.chen@windriver.com>
Date: Fri, 31 Mar 2023 16:02:50 +0800
Subject: [PATCH] Makefile.am: do not compile the code which was generated from
 test-full.proto in protobuf-c-native

Those code was auto generated by protoc command with test-full.proto, those code are not compatible
with protobuf 4.22.x, so temporarily disable compile those code until protobuf-c upstream adapt
the test-full.proto with latest version protobuf.

Upstream-Status: Inappropriate [oe specific]

Signed-off-by: Xiangyu Chen <xiangyu.chen@windriver.com>
---
 Makefile.am | 116 ++++++++++++++++++++++++++--------------------------
 1 file changed, 58 insertions(+), 58 deletions(-)

diff --git a/Makefile.am b/Makefile.am
index 3b0d1d6..ff4c196 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -136,64 +136,64 @@ else
 
 LOG_COMPILER = $(VALGRIND)
 
-check_PROGRAMS += \
-	t/generated-code/test-generated-code \
-	t/generated-code2/test-generated-code2 \
-	t/version/version
-
-TESTS += \
-	t/generated-code/test-generated-code \
-	t/generated-code2/test-generated-code2 \
-	t/version/version
-
-t_generated_code_test_generated_code_SOURCES = \
-	t/generated-code/test-generated-code.c \
-	t/test.pb-c.c
-t_generated_code_test_generated_code_LDADD = \
-	protobuf-c/libprotobuf-c.la
-
-t_generated_code2_test_generated_code2_SOURCES = \
-	t/generated-code2/test-generated-code2.c \
-	t/test-full.pb-c.c \
-	t/test-optimized.pb-c.c
-t_generated_code2_test_generated_code2_LDADD = \
-	protobuf-c/libprotobuf-c.la
-
-noinst_PROGRAMS += \
-	t/generated-code2/cxx-generate-packed-data
-
-t_generated_code2_cxx_generate_packed_data_SOURCES = \
-	t/generated-code2/cxx-generate-packed-data.cc \
-	t/test-full.pb.cc \
-	protobuf-c/protobuf-c.pb.cc
-$(t_generated_code2_cxx_generate_packed_data_OBJECTS): t/test-full.pb.h
-t_generated_code2_cxx_generate_packed_data_CXXFLAGS = \
-	$(AM_CXXFLAGS) \
-	$(protobuf_CFLAGS)
-t_generated_code2_cxx_generate_packed_data_LDADD = \
-	$(protobuf_LIBS)
-
-t/test.pb-c.c t/test.pb-c.h: $(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test.proto
-	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test.proto
-
-t/test-optimized.pb-c.c t/test-optimized.pb-c.h: $(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test-optimized.proto
-	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test-optimized.proto
-
-t/test-full.pb-c.c t/test-full.pb-c.h: $(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test-full.proto
-	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test-full.proto
-
-t/test-full.pb.cc t/test-full.pb.h: @PROTOC@ $(top_srcdir)/t/test-full.proto
-	$(AM_V_GEN)@PROTOC@ -I$(top_srcdir) --cpp_out=$(top_builddir) $(top_srcdir)/t/test-full.proto
-
-t/generated-code2/test-full-cxx-output.inc: t/generated-code2/cxx-generate-packed-data$(EXEEXT)
-	$(AM_V_GEN)$(top_builddir)/t/generated-code2/cxx-generate-packed-data$(EXEEXT) > $(top_builddir)/t/generated-code2/test-full-cxx-output.inc
-
-BUILT_SOURCES += \
-	t/test.pb-c.c t/test.pb-c.h \
-	t/test-full.pb-c.c t/test-full.pb-c.h \
-	t/test-optimized.pb-c.c t/test-optimized.pb-c.h \
-	t/test-full.pb.cc t/test-full.pb.h \
-	t/generated-code2/test-full-cxx-output.inc
+# check_PROGRAMS += \
+#	t/generated-code/test-generated-code \
+#	t/generated-code2/test-generated-code2 \
+#	t/version/version
+
+# TESTS += \
+#	t/generated-code/test-generated-code \
+#	t/generated-code2/test-generated-code2 \
+#	t/version/version
+
+# t_generated_code_test_generated_code_SOURCES = \
+#	t/generated-code/test-generated-code.c \
+#	t/test.pb-c.c
+# t_generated_code_test_generated_code_LDADD = \
+#	protobuf-c/libprotobuf-c.la
+
+# t_generated_code2_test_generated_code2_SOURCES = \
+#	t/generated-code2/test-generated-code2.c \
+#	t/test-full.pb-c.c \
+#	t/test-optimized.pb-c.c
+# t_generated_code2_test_generated_code2_LDADD = \
+#	protobuf-c/libprotobuf-c.la
+
+# noinst_PROGRAMS += \
+#	t/generated-code2/cxx-generate-packed-data
+
+# t_generated_code2_cxx_generate_packed_data_SOURCES = \
+#	t/generated-code2/cxx-generate-packed-data.cc \
+#	t/test-full.pb.cc \
+#	protobuf-c/protobuf-c.pb.cc
+# $(t_generated_code2_cxx_generate_packed_data_OBJECTS): t/test-full.pb.h
+# t_generated_code2_cxx_generate_packed_data_CXXFLAGS = \
+#	$(AM_CXXFLAGS) \
+#	$(protobuf_CFLAGS)
+# t_generated_code2_cxx_generate_packed_data_LDADD = \
+#	$(protobuf_LIBS)
+
+# t/test.pb-c.c t/test.pb-c.h: $(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test.proto
+#	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test.proto
+
+# t/test-optimized.pb-c.c t/test-optimized.pb-c.h: $(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test-optimized.proto
+#	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test-optimized.proto
+
+# t/test-full.pb-c.c t/test-full.pb-c.h: $(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) $(top_srcdir)/t/test-full.proto
+#	$(AM_V_GEN)@PROTOC@ --plugin=protoc-gen-c=$(top_builddir)/protoc-c/protoc-gen-c$(EXEEXT) -I$(top_srcdir) --c_out=$(top_builddir) $(top_srcdir)/t/test-full.proto
+
+# t/test-full.pb.cc t/test-full.pb.h: @PROTOC@ $(top_srcdir)/t/test-full.proto
+#	$(AM_V_GEN)@PROTOC@ -I$(top_srcdir) --cpp_out=$(top_builddir) $(top_srcdir)/t/test-full.proto
+
+# t/generated-code2/test-full-cxx-output.inc: t/generated-code2/cxx-generate-packed-data$(EXEEXT)
+#	$(AM_V_GEN)$(top_builddir)/t/generated-code2/cxx-generate-packed-data$(EXEEXT) > $(top_builddir)/t/generated-code2/test-full-cxx-output.inc
+
+# BUILT_SOURCES += \
+#	t/test.pb-c.c t/test.pb-c.h \
+#	t/test-full.pb-c.c t/test-full.pb-c.h \
+#	t/test-optimized.pb-c.c t/test-optimized.pb-c.h \
+#	t/test-full.pb.cc t/test-full.pb.h \
+#	t/generated-code2/test-full-cxx-output.inc
 
 if BUILD_PROTO3
 
-- 
2.34.1

